lib/matplotlib/backends/backend_pgf.py

```diff
@@ -779,15 +763,20 @@ class GraphicsContextPgf(GraphicsContextBase):
 
 
 class TmpDirCleaner:
-    remaining_tmpdirs = set()
+    _remaining_tmpdirs = set()
```

Comment:
This change was only for the deprecation, so the renaming isn't really needed here.